articles/quickstart/native/ios-objc/05-authorization.md

```diff
@@ -13,29 +13,36 @@ contentType: tutorial
 useCase: quickstart
 ---
 
-Many identity providers supply access claims which contain, for example, user roles or groups. You can request the access claims in your token with `scope: openid roles` or `scope: openid groups`.
+Many identity providers supply access claims which contain, for example, user roles or groups. You can request the access claims in your token with `showLoginWithScope:@"openid roles"` or `showLoginWithScope:@"openid groups"`.
 
-If an identity provider does not supply this information, you can create a rule for assigning roles to users. 
+If an identity provider does not supply this information, you can create a rule for assigning roles to users.
```

Comment:
Let's add a link in here to direct readers to read more about Rules and how they can apply them. Something like:

```suggestion
If an identity provider does not supply this information, you can [create a Rule](https://auth0.com/docs/rules)  for assigning roles to users.
```